samples/sensor/bmm150/README.rst

```diff
@@ -0,0 +1,8 @@
+BMM150 GeoMagnetic Sensor
```

Comment:
Please use the example template for sample code found in your cloned zephyr tree: doc/templates/sample.tmpl